lib/main_ltp.pm

```diff
@@ -63,21 +63,10 @@ sub load_kernel_tests {
             loadtest_kernel 'update_kernel';
         }
 
-        # transactional needs to first run install_ltp due broken grub menu
-        # counting detection in add_custom_grub_entries():
-        # Test died: Unexpected number of grub entries: 5, expected: 3 at lib/bootloader_setup.pm line 166.
```

Comment:
I don't understand this part at all. Lets see how it will behave with `update_kernel`.